### PR TITLE
Contrib site

### PR DESCRIPTION
**Pull request by whmii**
screencapture-localhost-8000-1448658501126

### CODE REVIEW

**Comment by whmii** [comment]
:dog: :gun:

File: contrib/index.html [hunk 1]
```diff
@@ -0,0 +1,199 @@
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <title>Have fun ♥️</title>
```
Comment (joshuaogle) [inline contrib/index.html]: Is there an ASCII something we can use to make that more apparent in the code? It's just a rectangle here on Github for instance

File: contrib/index.html [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+      </header>
+      <main role="main">
+        <h3>Basic</h3>
+        <section class="page-section zero">
+        <code>@include outer-container;</code>
+          <div class="page-section__inner zero__alpha">
+            <code>@include span-columns(6);</code>
+          </div>
+          <div class="page-section__inner zero__beta">
+            <code>@include span-columns(6);<br></code>
```
Comment (joshuaogle) [inline contrib/index.html]: We don't need the `<br>` here, do we?

File: contrib/index.html (hunk 1 above)
Comment (whmii) [inline contrib/index.html]: I can switch to ascii

Comment (whmii) [inline contrib/index.html]: It's just…
screen shot 2015-11-27 at 4 18 22 pm

File: contrib/styles.scss [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+$visual-grid-color: #e6f6ff;
+$visual-grid-opacity: 0.4;
+
+// Colors
+$white: #fafafa;
+$lightblue: #d5eefb;
+$blue: #aadbf5;
+$darkblue: #41a8df;
+$darkerblue: #298ec3;
+$darkerblue: #072f44;
```
Comment (joshuaogle) [inline contrib/styles.scss]: Duplicate variables here, so only the second one is used

File: contrib/styles.scss [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+.welcome-message {
+  @include clearfix();
+  @include padding($base-spacing null);
+  background-color: $blue;
+  margin-bottom: 1.5rem;
+  text-align: center;
+}
+
+.welcome-message-title {
+  font-family: "helvetica Neue", "Helvetica", sans-serif;
```
Comment (joshuaogle) [inline contrib/styles.scss]: This isn't case-sensitive, but it would look better as "**H**elvetica Neue"

File: contrib/styles.scss [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+.third__alpha {
+  @include fill-parent();
+  @include row(table);
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.third__alpha__alpha {
+  @include span-columns(4);
+  border-right: 1px solid rgba(255,255,255, 0.5);
```
Comment (joshuaogle) [inline contrib/styles.scss]: Can we put some spaces between the commas or just use `rgba(#fff, 0.5)`?

File: contrib/styles.scss [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+  @include padding($base-spacing null);
+  background-color: $blue;
+  margin-bottom: 1.5rem;
+  text-align: center;
+}
+
+.welcome-message-title {
+  font-family: "helvetica Neue", "Helvetica", sans-serif;
+  font-weight: 200;
+  line-height: $base-line-height;
```
Comment (joshuaogle) [inline contrib/styles.scss]: This line height looks a little too big for a title

File: contrib/index.html [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+          <code>@include outer-container;</code>
+        </div>
+        <section class="page-section fifth">
+          <div class="box">
+<pre>
+  .box {
+    @include span-columns(3);
+    @include omega(4n);
+  }
+</pre>
```
Comment (joshuaogle) [inline contrib/index.html]: This part looks really strange centered, but I'm not sure I have a good solution.

**Comment by joshuaogle** [comment]
Personally, I'm with Hound and would prefer keeping things simple with hyphens rather than confusing people with BEM.

This look great so far though and I'll be really happy to have it in there for contributors. Good work, @whmii!

File: Gulpfile.js [hunk 8]
```diff
@@ -0,0 +1,33 @@
+var gulp = require("gulp"),
+  autoprefix = require("gulp-autoprefixer"),
+  sass = require("gulp-sass"),
```
Comment (whmii) [inline Gulpfile.js]: node

File: Gulpfile.js [hunk 9]
```diff
@@ -0,0 +1,33 @@
+var gulp = require("gulp"),
+  autoprefix = require("gulp-autoprefixer"),
```
Comment (whmii) [inline Gulpfile.js]: node

File: Gulpfile.js [hunk 10]
```diff
@@ -0,0 +1,33 @@
+var gulp = require("gulp"),
```
Comment (whmii) [inline Gulpfile.js]: node

File: Gulpfile.js [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+  autoprefix = require("gulp-autoprefixer"),
+  sass = require("gulp-sass"),
+  connect = require("gulp-connect"),
```
Comment (whmii) [inline Gulpfile.js]: node

File: Gulpfile.js [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+  bourbon = require("node-bourbon").includePaths;
```
Comment (whmii) [inline Gulpfile.js]: node

**Comment by whmii** [comment]
@joshuaogle thanks for the feedback :+1:

**Pull request merged**